autometa/common/external/hmmer.py

```diff
@@ -234,11 +238,33 @@ def translater(x): return translations.get(x, x.rsplit('_', 1)[0])
     return outfpath
 
 
-def main(args):
+def main():
+    import argparse
+    import logging as logger
+    logger.basicConfig(
+        format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
+        datefmt='%m/%d/%Y %I:%M:%S %p')
+    parser = argparse.ArgumentParser(
+        description='Retrieves markers with provided input assembly')
+    parser.add_argument('orfs', help='</path/to/assembly.orfs.faa>')
+    parser.add_argument('hmmdb', help='</path/to/hmmpressed/hmmdb>')
+    parser.add_argument('cutoffs', help='</path/to/hmm/cutoffs.tsv>')
+    parser.add_argument('hmmscan', help='</path/to/hmmscan.out>')
+    parser.add_argument('markers', help='</path/to/markers.tsv>')
+    parser.add_argument('--log', help='</path/to/parallel.log>')
+    parser.add_argument('--force', help="force overwrite of out filepath",
+                        action='store_true')
+    parser.add_argument('--cpus', help='num cpus to use', default=0, type=int)
+    parser.add_argument(
+        '--parallel', help="enable GNU parallel", action='store_true')
+    parser.add_argument('--verbose', help="add verbosity", action='store_true')
+    args = parser.parse_args()
+
     if args.verbose:
         logger.setLevel(logger.DEBUG)
-
-    try:
+    if os.path.exists(args.hmmscan) and os.stat(args.hmmscan).st_size > 0 and not args.force:
```

Comment:
This looks like it will resolve the issue. I've noticed we can also get the file size with `os.path.getsize(fpath)` which I think is a bit more intuitive than `os.stat(fpath).st_size`. Minor nitpick, but I recently came across this and have been changing this as I come across it. I think this occurs a few times in this script. So feel free to update throughout.

```suggestion
    if os.path.exists(args.hmmscan) and os.path.getsize(args.hmmscan) > 0 and not args.force:
```